app/database/migrations/20201201151227_create-jobs-table.js

```diff
@@ -0,0 +1,15 @@
+
+exports.up = function(knex) {
+    return knex.schema.createTable('jobs', tbl => {
+        tbl.increments('id').unique().nonNullable();
+        tbl.integer('class_id').unsigned().nonNullable();
+        tbl.timestamps(true, true); // creates 'created_at' and 'update_at' columns (useTimestamps = true, defaultToNow = true)
+        tbl.datetime('scheduled_time');
+        // TODO: @Arnav add status(enum)
```

Comment:
You can take off this comment, Arnav won't be making his changes in this file, he will be making a separate migration file.

Reply:
Sure thing!